src/pages/dashboard/monitor/index.tsx

```diff
@@ -0,0 +1,272 @@
+import { Card, Col, Row, Statistic, Tooltip } from 'antd';
+import { FormattedMessage, formatMessage } from 'umi-plugin-react/locale';
+import React, { Component } from 'react';
+
+import { Dispatch } from 'redux';
+import { GridContent } from '@ant-design/pro-layout';
+import { connect } from 'dva';
+import numeral from 'numeral';
+import { StateType } from './model';
+import { Pie, WaterWave, Gauge, TagCloud } from './components/Charts';
+import ActiveChart from './components/ActiveChart';
+import styles from './style.less';
+
+const { Countdown } = Statistic;
+
+const targetTime = new Date().getTime() + 3900000;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['magic numbers' are not allowed: 3900000](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=4583897)